README.md

```diff
@@ -519,28 +538,33 @@ for f in `ls -1 import/*.conf`; do f=`basename ${f}`; diff import/${f} ~/src/owa
 
 If there are no differences, then the rulesets are the same.
 
-### Module Tests
+### Module Examples
 
-The test files also show how this module works, and are a helpful reference if you wish to extend this module.
+The examples files also show how this module works, and are a helpful reference if you wish to extend this module.
 
 Looking at `examples/test_lexer.py`, the following command will show you how and what tokens are found in your config:
 
-```
+```bash
 ./examples/test_lexer.py /path/to/owasp-modsecurity-crs/rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf debug
 ```
 
 Run the following command to see how the parser applies language rules to the tokens:
 
-```
-./examples/test_parser.py /path/to/owasp-modsecurity-crs/rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf debug
+```bash
+./examples/parse.py /path/to/owasp-modsecurity-crs/rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf debug
```

Comment:
The mentioned file (`examples/parse.py`) doesn't exists, only the `test_parser`. The parser and lexer are two different steps, should be keep both examples.

Reply:
Done.